includes/REST_API/Products_Controller.php

```diff
@@ -314,7 +314,8 @@ protected function get_items_shopify( $request ) {
 				'productBrand'         => $product['vendor'],
 				// TODO: Maybe eventually provide full price range.
 				// See https://github.com/ampproject/amphtml/issues/37957.
-				'productPrice'         => $product['priceRange']['minVariantPrice']['amount'],
+				// TODO: Replace hardcoded value with real price as a number (AMP schema validation).
+				'productPrice'         => 10,
```

Comment:
Why this change instead of type casting or something?

Reply:
Testing ...  if this passes with fix that up.

Reply:
Successful run here https://github.com/GoogleForCreators/web-stories-wp/actions/runs/2229993362

Updated to cast as a `float` for now.